pkg/controller/cassandracluster/pod_test.go

```diff
@@ -82,16 +83,48 @@ func TestGetLastOrFirstPod(t *testing.T) {
 	podlist = mkList(mkPod(10, true), mkPod(1, false))
 
 	pod, err = GetLastOrFirstPod(podlist, first)
+	if pod.Status.Phase != v1.PodRunning || pod.DeletionTimestamp != nil {
+		err = fmt.Errorf("Pod is not running")
+	}
 	assertPodIsNotRunning(t, err)
 
 	podlist.Items[0].Status = v1.PodStatus{Phase: v1.PodRunning}
 	ts := metav1.Now()
 	podlist.Items[0].DeletionTimestamp = &ts
 
 	pod, err = GetLastOrFirstPod(podlist, first)
+	if pod.Status.Phase != v1.PodRunning || pod.DeletionTimestamp != nil {
+		err = fmt.Errorf("Pod is not running")
+	}
 	assertPodIsNotRunning(t, err)
 }
 
+
+func TestAllPodContainerReady(t *testing.T) {
+
+	mkPod := func(id int, containers ...bool) *v1.Pod {
+		containerStatuses := []v1.ContainerStatus{}
+	var i int = 0
```

Comment:
let's use a better name for that variable and indent the code properly

Reply:
I took that name from your code :)